source/faq/faq.rst

```diff
@@ -101,20 +96,21 @@ How can I update my deploy to dojot's latest version?
 You need to follow some steps:
 
 1. Update the docker-compose repository to the latest version.
-	
-.. code-block:: bash	
-	
-	$ cd <path-to-your-clone-of-docker-compose>
-	$ git checkout master && git pull
-			
+
+.. code-block:: bash
+
+  $ cd <path-to-your-clone-of-docker-compose>
+  $ git checkout master && git pull
+
 2. Deploy the latest docker images.
 
 .. code-block:: bash
-	
-	$ docker-compose pull && docker-compose up -d --build
-	
-This procedure also applies to the available virtual machines once they do use docker-compose.
-	
+
+  $ docker-compose pull && docker-compose up -d --build
```

Comment:
Those docker commands may benefit from that "you may need sudo" note

Reply:
Ok